dev-app/src/screens/RegisterInternetReaderScreen.tsx

```diff
@@ -91,6 +91,7 @@ export default function RegisterInternetReaderScreen() {
           onPress={() => {
             navigation.navigate('LocationListScreen', {
               onSelect: (location: Location) => setSelectedLocation(location),
+              showDummyLocation: true,
```

Comment:
Looks like we never pass false for `showDummyLocation`. In that case can we remove the boolean and just always display the locations?

Reply:
Yep, the main consideration before was that in other scenarios, such as when entering from the Discover Reader menu, it was not necessary to display Dummy Locations for the user to select.

Reply:
Ah gotcha, so we do render the `LocationListScreen` without dummy locations. Thanks!